docs/api/tracker.rst

```diff
@@ -7,6 +7,21 @@
 Tracker
 =======
 
+.. edit-link::
+
+Trackers maintain the state of a dialogue between the assistant and the user in the form
+of conversation sessions. The default conversation session configuration looks as
+follows:
+
+.. code-block:: yaml
+
+  session_config:
```

Comment:
I'd probably leave out the example here and just link to `session_config`. Makes it a bit DRYer

Reply:
hm, just took @erohmensing's suggestion to have it in both. I'm also leaning towards not having it in here, but don't mind

Reply:
Ok, then consider it optional from my side

Reply:
@wochinge wdym by DRYer? I'm not 100p attached to explaining the defaults here, but think having an example of what it looks like in the domain is helpful, for those who are less comfortable with "put these 2 keys under this key in the domain" -- I would see people doing it in maybe list format for example, since so much other stuff in the domain is. 

But if both of you agree, it can also be optional from my side 😃

Reply:
👍